Backpack-SwiftUI/GraphicPromo/Classes/BPKGraphicPromo.swift

```diff
@@ -70,6 +70,29 @@ public struct BPKGraphicPromo: View {
         self.verticalAlignment = verticalAlignment
     }
     
+    public init(
```

Comment:
Probably the most significant breaking change. We want a direct init for the sponsored version, as some of the fields from the organic version are not used in the sponsored one.